scripts/src/models/translations.ts

```diff
@@ -30,13 +30,13 @@ export class OfferTranslation extends BaseEntity {
 		}
 
 		const query = OfferTranslation.createQueryBuilder("translations");
-		if (languages) {
+		if (languages && !languages.length) {
```

Comment:
I think you mean `languages.length` without the `!` or `!!languages.length`

Reply:
Yep, thanks